browser-test/src/admin/admin_predicates.test.ts

```diff
@@ -1214,18 +1216,22 @@ test.describe('create and edit predicates', () => {
       )
       // Validate that ineligible page is accessible.
       await validateAccessibility(page)
-      await page.goBack()
 
-      await applicantQuestions.clickReview()
```

Comment:
Now that the "Review" button saves answers, if you click "Review" on a page that has ineligible answers then you'll be taken to the final "Based on your responses, you may not qualify" page. However, this test wants to test that the review page has the "you may not qualify" toast.

The "Go back and edit" button on the "Based on your responses, you may not qualify" page will always take you to the review page, so I updated this test to click that button so we can still get a screenshot of the review page with the "you may not qualify" toast.

Reply:
Thanks for the explaination!